PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/build.gradle

```diff
@@ -10,5 +10,5 @@ dependencies {
     implementation 'com.microsoft.azure:azure-toolkit-common-lib'
     implementation 'com.microsoft.azure:azure-toolkit-ide-common-lib'
     aspect 'com.microsoft.azure:azure-toolkit-common-lib'
-
+    implementation group: 'org.json', name: 'json', version: '20240303'
```

Comment:
Maybe if we don't specify a version it will use the latest one? Why do the packages above not have a version number?

Reply:
there's build errors if the version is not specified